docs/storage.md

```diff
@@ -89,6 +89,7 @@ config:
   trace:
     enable: false
   list_objects_version: ""
+  dns_style: false
```

Comment:
This makes sense, just can we:

* Find a more specific name?
* Use enums instead of boolean - it will "scale" better if there will be a 3rd option.

For example:

```suggestion
  dns_style: "virtual-hosted" # Or `"path" read more about it here https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html
```

Reply:
As requested we changed the name and are using enums.
If

Reply:
The CI/test are fixed now.